client/src/components/Header/components/exam-nav.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import FreeCodeCampLogo from '../../../assets/icons/freecodecamp';
+import MicrosoftLogo from '../../../assets/icons/microsoft-logo';
+
+import './exam-nav.css';
+
+const ExamNav = (): JSX.Element => {
+  const { t } = useTranslation();
+
+  return (
+    <nav aria-label={t('aria.primary-nav')} className='exam-nav' id='exam-nav'>
+      <div className='logo freecodecamp-logo'>
+        <FreeCodeCampLogo aria-hidden='true' />
+      </div>
+      <div className='logo partner-logo'>
+        <MicrosoftLogo aria-hidden='true' />
```

Comment:
Do we want to make this a prop instead? So that we can dynamically select the icon (I imagine we'll be adding more exams that aren't microsoft specific)

Reply:
Yes, that's the better way to go - but I just went with it like this for now since we only have the one exam.